[quant][graphmode] Enable foldbn tests

jerryzh168 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29220 [quant][graphmode] Enable foldbn tests**
* #29219 [jit] Expose class constant through `attr` and `setattr` in object
* #30787 [jit] Add unsafeRemoveConstant for ClassType
* #31012 [jit] Fix getConstant
* #29218 [jit] Preserve constant from ConcreteModuleType to ClassType

Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

jerryzh168 committed:
[quant][graphmode] Enable foldbn tests

Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
Looks good! I would change the commit message to something like "fix FoldConvBN pass" though (or something similar but more specific) - that's the real change in this PR.

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

kostmo commented:
## CircleCI build failures summary
As of commit 4feb4a4:
* **1/1** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=4feb4a451e3a1dce88321f38f7313ca766251ba7).

### 1 new failure recognized by patterns
The following build failures don't appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3919620) pytorch_linux_xenial_py3_clang5_mobile_build (1/1)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=72958185))
```
Dec 12 06:03:09        ^
Dec 12 06:03:09 /var/lib/jenkins/workspace/torch/csrc/jit/script/class_type.cpp:143:3: note: in expansion of macro 'TORCH_INTERNAL_ASSERT'
Dec 12 06:03:09    TORCH_INTERNAL_ASSERT(constantNames_.size() == constantValues_.size());
Dec 12 06:03:09    ^
Dec 12 06:03:09 /var/lib/jenkins/workspace/torch/csrc/jit/script/class_type.cpp:145:24: error: 'constantNames_' was not declared in this scope
Dec 12 06:03:09    for (const auto& c : constantNames_) {
Dec 12 06:03:09                         ^
Dec 12 06:03:09 /var/lib/jenkins/workspace/torch/csrc/jit/script/class_type.cpp:152:14: error: 'constantNames_' was not declared in this scope
Dec 12 06:03:09    if (pos >= constantNames_.size()) {
Dec 12 06:03:09               ^
Dec 12 06:03:09 /var/lib/jenkins/workspace/torch/csrc/jit/script/class_type.cpp:155:10: error: 'constantValues_' was not declared in this scope
Dec 12 06:03:09    return constantValues_[pos];
Dec 12 06:03:09           ^
Dec 12 06:03:09 make[2]: *** [caffe2/CMakeFiles/torch_cpu.dir/__/torch/csrc/jit/script/class_type.cpp.o] Error 1
Dec 12 06:03:09 make[2]: *** Waiting for unfinished jobs....
Dec 12 06:03:09 caffe2/CMakeFiles/torch_cpu.dir/build.make:8897: recipe for target 'caffe2/CMakeFiles/torch_cpu.dir/__/torch/csrc/jit/script/class_type.cpp.o' failed
Dec 12 06:03:13 CMakeFiles/Makefile2:882: recipe for target 'caffe2/CMakeFiles/torch_cpu.dir/all' failed
Dec 12 06:03:13 make[1]: *** [caffe2/CMakeFiles/torch_cpu.dir/all] Error 2
Dec 12 06:03:13 make: *** [all] Error 2
Dec 12 06:03:13 Makefile:138: recipe for target 'all' failed
Dec 12 06:03:13 + sccache_epilogue
```


---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]

jerryzh168 committed:
Update on "[quant][graphmode] Enable foldbn tests"


Summary:
Support for accessing constant is added in previous
PRs, this PR re-enables the foldbn tests

Test Plan:
test_jit.py

Reviewers:
mvz

Subscribers:

Tasks:

Tags:

Differential Revision: [D18846848](https://our.internmc.facebook.com/intern/diff/D18846848)

[ghstack-poisoned]
